rocksdb/db.go

```diff
@@ -69,7 +69,11 @@ func (db *RocksDB) Get(key []byte) ([]byte, error) {
 	if len(key) == 0 {
 		return nil, tmdb.ErrKeyEmpty
 	}
-	return db.db.GetBytes(db.ro, key)
```

Comment:
I don't think this change is necessary as db.db.GetBytes() does the right thing in one pass, better than db.db.Get() -> moveSliceToBytes().

Reply:
It seems to be a somewhat dependent code on rocksdb implementation, but there is no big problem, so I will correct it.